genesisSmartContracts.json

```diff
@@ -1,14 +1,14 @@
 [
   {
-    "owner": "erd1nszf4y8dkply45skqxa2998uypc53tdsd26ycl5yndtqhl7j2tssfxzk2y",
+    "owner": "erd1vxy22x0fj4zv6hktmydg8vpfh6euv02cz4yg0aaws6rrad5a5awqgqky80",
     "filename": "./config/genesisContracts/delegation.wasm",
     "vm-type": "0500",
     "init-parameters": "%validator_sc_address%@03E8@00@030D40@030D40",
     "type": "delegation",
     "version": "0.4.*"
   },
   {
-    "owner": "erd1qfxe879dxy632pdeuq2f9xvvzlsnx2n5lhtjalyef833ff7s9nvqmgy727",
+    "owner": "erd188anxz35atlef7cucszypmvx88lhz4m7a7t7lhcwt6sfphpsqlkswfhcx2",
     "filename": "./config/genesisContracts/dns.wasm",
```

Comment:
Different from testnet. All good?

https://github.com/multiversx/mx-chain-testnet-config/blob/master/genesisSmartContracts.json

Reply:
not good, forgot to revert. Thanks & good catch :+1: